eureka-webapp/src/main/webapp/assets/js/eureka.job.js

```diff
@@ -46,6 +46,15 @@ window.eureka.job = new function () {
 						$('#destinationConfig').text(data.destinationId);
 						$('#startedDate').text(data.startedDateFormatted);
 						$('#finishedDate').text(data.finishedDateFormatted);
+                                                if(data.status === "Completed" && data.getStatisticsSupported){
+                                                    var browseOutputLink='';
+                                                    browseOutputLink = '<a href="'+$('#getStatisticsSupported').attr('contextPath')+'/protected/jobstats?jobId='+data.jobId+'">Browse Output</a>';                                                 
+                                                    $('#getStatisticsSupported').html(browseOutputLink).show();
+
+                                                }else{
+                                                    $('#getStatisticsSupported').hide();
+                                                }
+                                                
```

Comment:
Does this code show the destination's configured links in addition to the browse output link? It should display both. See job_submission.jsp.